packages/main/test/pageobjects/DayPickerTestPage.js

```diff
@@ -0,0 +1,27 @@
+class DayPickerTestHelper {
+	set id(id) {
+		this._sut = id;
+	}
+
+	get getDayPickerRoot() {
+		return browser.execute((id) => {
+			return document.getElementById(id).shadowRoot.querySelector("ui5-popover") // ui5-popover
+				.shadowRoot.querySelector("slot").assignedNodes()[0] // ui5-calendar
```

Comment:
Here and below, the calendar discovery can be simplified with $("ui5-calendar"), instead of .shadowRoot.querySelector("slot").assignedNodes()[0], as it is in the light DOM